### PR TITLE
Add image verification for HTML tags too

### PR DESCRIPTION
**Pull request by KrauseFx**
This will also validate local images referenced from an `img` tag

### CODE REVIEW

**Comment by KrauseFx** [comment]
Oh yeah, obviously this fails until https://github.com/fastlane/docs/pull/45 got merged

File: fastlane/Fastfile [hunk 1]
```diff
@@ -31,7 +31,8 @@ private_lane :ensure_image_assets do
   errors = []
   Dir["docs/**/*.md"].each do |current_file|
     content = File.read(current_file)
-    matches = content.scan(/\!\[.*\]\((.*\.png)\)/)
+    matches = content.scan(/\!\[.*\]\((.*\.png)\)/) # markdown matches
+    matches += content.scan(/\<img.*src="(.*\.png)/) # HTML matches
```
Comment (mfurtak) [inline fastlane/Fastfile]: Might eventually want to match .jpg? Others?

Otherwise :+1:

Comment (KrauseFx) [inline fastlane/Fastfile]: Yeah, I'll do a separate PR with this 👍

Comment (KrauseFx) [inline fastlane/Fastfile]: .gifs too

Comment (mfurtak) [inline fastlane/Fastfile]: yes-signs-loop

**Pull request merged**